src/pluginGlobals.ts

```diff
@@ -138,6 +138,14 @@ export function popFileArgs(filename: string): SeatbeltArgs {
     return args
   }
 
+  if (lastLintedFile?.filename === filename) {
```

Comment:
When running eslint runs this plugin, it will first call `configure.create(context)` in `configure.ts` which loads the plugin config and stores it in `temporaryFileArgs`.

However, when I provoce a fatal TS error with invalid syntax, I noticed that the config is not re-initialized but it only calls the postprocessor again. As a result, during those runs, it doesn't pick up any configuration.

I don't understand the reason for "popping" rules at all (as well as I don't understand the eslint development environment in depth). However, it sounds reasonable to fallback to the arguments from the last run if there are no arguments available.